arlo_server/__init__.py

```diff
@@ -26,51 +27,59 @@ def create_election(election_id=None):
     db.session.commit()
     return election_id
 
+
 def init_db():
     db.create_all()
 
+
 def get_election(election_id):
-    return Election.query.filter_by(id = election_id).one()
+    return Election.query.filter_by(id=election_id).one()
+
 
 def contest_status(election):
     contests = {}
 
     for contest in election.contests:
-        contests[contest.id] = dict([
-            [choice.id, choice.num_votes]
-            for choice in contest.choices])
+        contests[contest.id] = dict([[choice.id, choice.num_votes]
+                                     for choice in contest.choices])
         contests[contest.id]['ballots'] = contest.total_ballots_cast
         contests[contest.id]['numWinners'] = contest.num_winners
         contests[contest.id]['votesAllowed'] = contest.votes_allowed
 
     return contests
 
+
 def sample_results(election):
     contests = {}
 
     for contest in election.contests:
-        contests[contest.id] = dict([
-            [choice.id, 0]
-            for choice in contest.choices])
+        contests[contest.id] = dict([[choice.id, 0]
+                                     for choice in contest.choices])
 
-        round_contests = RoundContest.query.filter_by(contest_id = contest.id).order_by('round_id').all()
+        round_contests = RoundContest.query.filter_by(
+            contest_id=contest.id).order_by('round_id').all()
         for round_contest in round_contests:
             for result in round_contest.results:
-                contests[contest.id][result.targeted_contest_choice_id] += result.result
+                contests[contest.id][
+                    result.targeted_contest_choice_id] += result.result
```

Comment:
Any idea if there's some way to get it to not break up brackets like this? I'm not seeing anything specific in PEP8 about it, is it pythonic?